package.json

```diff
@@ -42,48 +43,59 @@
     "@babel/plugin-transform-parameters": "^7.10.4",
     "@babel/plugin-transform-runtime": "^7.10.4",
     "@babel/plugin-transform-typescript": "^7.15.0",
+    "@babel/plugin-proposal-json-strings": "^7.18.6",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.15.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.15.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.15.0",
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.15.0",
     "@cyclonedx/cyclonedx-npm": "^1.11.0",
-    "@types/node": "^15.12.2",
-    "@typescript-eslint/eslint-plugin": "^4.29.0",
-    "@typescript-eslint/parser": "^4.29.0",
-    "babel-jest": "^27.0.6",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@types/jest": "^29.1.2",
+    "@types/node": "^18.11.0",
+    "@typescript-eslint/eslint-plugin": "^5.40.1",
+    "@typescript-eslint/parser": "^5.40.1",
+    "babel-jest": "^29.2.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "clean-webpack-plugin": "^4.0.0-alpha.0",
     "copy-webpack-plugin": "^9.0.1",
-    "eslint": "^7.32.0",
+    "eslint": "8.25.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jest": "^24.3.2",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-jest": "^27.1.2",
+    "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.24.0",
     "husky": "^7.0.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.0.6",
+    "jest": "^29.2.0",
+    "jest-environment-jsdom": "^29.0.3",
     "jest-junit": "^15.0.0",
-    "jest-prosemirror": "^1.0.3",
+    "jest-prosemirror": "^2.0.4",
     "jest-sonar-reporter": "^2.0.0",
     "lint-staged": "^11.1.1",
     "prettier": "^2.1.2",
     "stylelint": "^13.7.2",
     "stylelint-config-standard": "^22.0.0",
-    "ts-jest": "^27.0.4",
+    "ts-jest": "^29.0.3",
     "ts-node": "^10.2.0",
     "webpack": "^5.48.0",
     "webpack-cli": "^4.7.0",
     "write-file-webpack-plugin": "4.5.1"
   },
   "dependencies": {
```

Comment:
These should all be peerDependencies, with the exception of typescript should be a devDependency.